### ISSUE TITLE
FELIX: Flexible Text Editing Through Tagging and Insertion

### ISSUE THREAD

**Issue by shermansiu** [huggingface/transformers#21122]
### Model description

FELIX is an encoder-only text editing model, which allows for faster editing and summarization than sequence-to-sequence models, because the summarization can be computed in parallel instead of autoregressively.

- [Blog](https://ai.googleblog.com/2021/05/introducing-felix-flexible-text-editing.html?hl=hr&m=1)
- [Paper](https://aclanthology.org/2020.findings-emnlp.111/)

### Open source status

- [X] The model implementation is available
- [ ] The model weights are available

### Provide useful links for the implementation

https://github.com/google-research/google-research/tree/master/felix

No weights are available, but code to train it is available. A component of FELIX is BERT, so training FELIX is a matter of fine-tuning a pre-trained BERT model.

@Jmallins

**Comment by shermansiu** [comment]
Because FELIX can be applied to any encoder-only model, perhaps what is needed is an `AutoModelForSequenceEditing`/`<ModelName>ModelForSequenceEditing`?

**Comment by NielsRogge** [comment]
Closing this as it's a duplicate of #11632
